### PR TITLE
Add netlify deploy

### PR DESCRIPTION
**Pull request by timja**

### CODE REVIEW

**Comment by halkeye** [comment]
Sweet ❤️

**Review by halkeye — APPROVED** [review]

**Comment by dduportal** [comment]
As soon as https://github.com/jenkins-infra/pipeline-library/pull/282 is merged, you can re-trigger the build here: that should work as expected

**Review by dduportal — APPROVED** [review]
LGTM (that will be manually updated but let's start with this one)

File: Dockerfile [hunk 1]
```diff
@@ -86,14 +96,15 @@ ARG JX_RELEASE_VERSION=2.5.1
 ARG JENKINS_AGENT_VERSION=4.11.2-2
 ARG ASDF_VERSION=0.8.1
 
-LABEL io.jenkins-infra.tools="img,container-structure-test,git,make,hadolint,gh,nodejs,npm,blobxfer,jx-release-version,jenkins-agent"
+LABEL io.jenkins-infra.tools="img,container-structure-test,git,make,hadolint,gh,nodejs,npm,blobxfer,jx-release-version,jenkins-agent,netlify-deploy"
```
Comment (dduportal) [inline Dockerfile:99]: Test are failing with the message `Error: label io.jenkins-infra.tools value img,container-structure-test,git,make,hadolint,gh,nodejs,npm,blobxfer,jx-release-version,jenkins-agent,netlify-deploy does not match expected value: img,container-structure-test,git,make,hadolint,gh,nodejs,npm,blobxfer,jx-release-version,jenkins-agent
`

can you update `cst.yml`?

**Review by dduportal — CHANGES_REQUESTED** [review]

**Review by dduportal — APPROVED** [review]
🚀

**Comment by timja** [comment]
Weird I thought I pushed a fix this morning

**Pull request merged**